index.js

```diff
@@ -13,6 +16,8 @@ module.exports = Documentify
 function Documentify (entry, html, opts) {
```

Comment:
Could we perhaps make the `opts` argument positional? So if `html` isn't provided, it just picks up on `opts` - that way there's no need for `null, opts` :tada:

Reply:
whoops, that email reply went to the wrong one. addressed this, thanks for the suggestion!